datastream-kafka-connector/src/main/java/com/linkedin/datastream/connectors/kafka/AbstractKafkaConnector.java

```diff
@@ -553,4 +564,12 @@ public AbstractKafkaBasedConnectorTask getConnectorTask() {
       return _task;
     }
   }
+
+  @Override
+  public List<BrooklinMetricInfo> getMetricInfos() {
+    List<BrooklinMetricInfo> metrics = new ArrayList<>();
+    metrics.add(new BrooklinMeterInfo(buildMetricName(NUM_TASK_RESTARTS)));
```

Comment:
Once you move to using the metrics prefix, you'll need to modify this. Plus you should use a regex for matching:

`<metrics prefix> + MetricsAware.KEY_REGEX + <metrics name>`